modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -1,50 +1,35 @@
 // Module included in the following assemblies:
 // TODO
 // * networking/TBD
+// * networking/load-balancing-openstack.adoc
 // For thinking and reviewing, adding to networking/load-balancing-openstack.adoc
 
 [id="nw-osp-configuring-external-load-balancer_{context}"]
 = Configuring an external load balancer
 
-You can configure a {product-title} cluster on {rh-openstack-first} to use an external load balancer.
+You can configure a {product-title} cluster on {rh-openstack-first} to use an external load balancer in place of the default load balancer.
 
 // Maybe an About mod in support
 
 .Prerequisites
 
-* On your load balancer, ports 6443, 443, and 80 must be available to any users of your system.
-* On your load balancer, port 22623, which is used to serve ignition start-up configurations to nodes, is not exposed outside of the cluster.
+* On your load balancer, TCP over ports 6443, 443, and 80 must be available to any users of your system.
 
-.Procedure
+** The {product-title} API uses port 6443. 
 
-// TODO: network formatting... style guide.
-. On your {rh-openstack} deployment, add floating IP addresses to all master nodes. For example:
-+
-[source,terminal]
-----
-$ openstack floating ip create --port master-port-0 <public_network>
-----
-+
-[source,terminal]
-----
-$ openstack floating ip create --port master-port-1 <public_network>
-----
-+
-[source,terminal]
-----
-$ openstack floating ip create --port master-port-2 <public_network>
-----
+** Cluster applications use ports 443 and 80.
 
-. Find the floating IP addresses that are associated with your nodes:
-+
-[source,terminal]
-----
-$ openstack server list
-----
+* On your load balancer, port 22623, which is used to serve ignition start-up configurations to nodes, is not exposed outside of the cluster.
+
+* Your load balancer must be able to access every machine in your cluster. Methods to allow this access include:
+** Creating the load balancer on a subnet that is in the cluster's network, and then attaching a router interface to the subnet from an `OpenShift-external-router` object.
```

Comment:
Would this be clearer for this case: "Attaching the load balancer to the machines subnet" ?

Reply:
Sure.